projects/ng-components-docs/src/app/examples/components/accordion/accordion.component.ts

```diff
@@ -0,0 +1,76 @@
+                    id="e9WTWunf5M"
+                />
+            </defs>
+            <g>
+                <g>
+                    <g>
+                        <use xlinkHref="#e9WTWunf5M" />
+                        <g>
+                            <use xlinkHref="#e9WTWunf5M" strokeWidth="1" />
+                        </g>
+                    </g>
+                </g>
+            </g>
+        </svg>
+    `;
+    angleDoubleRightIcon: string = `
+        <svg name="angle-double-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512">
+            <path d="M166.9 264.5l-117.8 116c-4.7 4.7-12.3 4.7-17 0l-7.1-7.1c-4.7-4.7-4.7-12.3 0-17L127.3 256 25.1 155.6c-4.7-4.7-4.7-12.3 0-17l7.1-7.1c4.7-4.7 12.3-4.7 17 0l117.8 116c4.6 4.7 4.6 12.3-.1 17zm128-17l-117.8-116c-4.7-4.7-12.3-4.7-17 0l-7.1 7.1c-4.7 4.7-4.7 12.3 0 17L255.3 256 153.1 356.4c-4.7 4.7-4.7 12.3 0 17l7.1 7.1c4.7 4.7 12.3 4.7 17 0l117.8-116c4.6-4.7 4.6-12.3-.1-17z" />
+        </svg>
+    `;
+
+    accordionList: Array<AccrodionListItem> = [
+        {
+            header: "Accordion List Item 1",
+            subHeaderText: "Accordion Sub Header",
+            content: {
+                title: "Tempor incididun",
+                desc:
+                    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, " +
+                    "sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Morbi tristique senectus et netus. Lectus mauris ultrices eros in cursus turpis massa tincidunt.",
+            },
+        },
+        {
+            header: "Accordion List Item 2",
+            content: [
+                {
+                    title: "Excepteur sint",
+                    desc:
+                        "Vitae suscipit tellus mauris a diam maecenas sed. Feugiat in fermentum posuere urna nec tincidunt praesent semper." +
+                        "Tellus id interdum velit laoreet id donec. Morbi enim nunc faucibus a pellentesque sit. Vitae congue mauris rhoncus aenean.",
+                },
+                {
+                    title: "Duis aute",
+                    desc:
+                        "Eleifend donec pretium vulputate sapien nec sagittis. Malesuada fames ac turpis egestas." +
+                        "Molestie ac feugiat sed lectus vestibulum mattis. Suscipit adipiscing bibendum est ultricies integer quis auctor elit sed.",
+                },
+            ],
+        },
+        {
+            header: "Accordion List Item 3",
+            content: `
+            <div>
+                <p className="m-0">Ut nemo corporis inventore neque qui. Est quos facere et id praesentium ut in iusto qui. Labore vel est ab.</p>
+                <a href="https://seb.se" target="_blank">
+                    Link to seb.se
+                </a>
+            </div>`,
+        },
+    ];
```

Comment:
You can utilize the package `lorem-ipsum` to generate text like this and clean up the UI
given you have this:

```typescript
import { loremIpsum } from "lorem-ipsum";
```

You can do this:

```typescript
accordionList: Array<AccordionListItem> = Array(3).fill({
    header: loremIpsum({ units: "words", count: 3 }),
    subHeaderText: loremIpsum({ units: "words", count: 3 }),
    content: {
        title: loremIpsum({ units: "words", count: 3 }),
        desc: loremIpsum({ units: "paragraph" })
    },
});
```